### ISSUE TITLE
Basic support not updated by this tool?

### ISSUE THREAD

**Issue by foolip** [mdittmer/mdn-confluence#28]
It appears as though this tool cannot detect the presence of a constructor like `Notification` to inform the "Basic support" row in MDN compat tables. @mdittmer is that right, a consequence of the interface+api pair model of Confluence?

**Comment by foolip** [comment]
https://github.com/mdn/browser-compat-data/pull/3326 done manually to work around this.

**Comment by mdittmer** [comment]
@foolip that is correct. Confluence only deals in interface+API pairs. An "empty interface" will never be detected by Confluence. It would be possible to add a pass or check that looks for "any APIs for interface named _name_", but that would be a guess at "basic support".

**Comment by foolip** [comment]
Right, so we could infer basic support from any API on the interface being detected. That would probably be accurate, as there are very few empty interfaces out there...

Do you know off hand whereabouts in the code that would have to be added?

**Comment by mdittmer** [comment]
Don't recall. I'd have to dive into the code after months away from it.
(which I could do, but I don't know off hand).

A good first step might be to grep for distinctive JSON keys used in the
BCD a schema. `__compat` or something like that?
